en/components/general/update_guide.md

```diff
@@ -44,6 +44,17 @@ Unfortunately not all changes can be automatically updated. Changes bellow are s
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+### From 7.3.4 to 8.0.x
+* While updating, if you face the folllowing error `Package "@angular/compiler-cli" has an incompatible peer dependency to "typescript" (requires ">=3.1.1 <3.3", would install "3.4.5").`, you should update `@angular/core` package first. This is related to this known [Angular CLI issue](https://github.com/angular/angular-cli/issues/13095)
```

Comment:
Not sure if other encountered this, but I did:
![image](https://user-images.githubusercontent.com/3198469/59784456-24819800-92cb-11e9-9ed0-7f26ed73c392.png)
This is after you update `@angular/core` + `@angular/cli` which sets the web-animations-js dep to the special PR version and it seems the version compare fails. Can't find an issue for that, thought the pr thing is supposed to be temporary.

If you can also reproduce this, you can add a note about it and suggest `--force` for that update.